website/docs/reference/node-selection/set-operators.md

```diff
@@ -7,6 +7,11 @@ Providing multiple space-delineated arguments to the `--models`, `--exclude`, or
 the union of them all. If a resource is included in at least one selector, it will be 
 included in the final set.
 
+Run all ancestors of snowplow_sessions and fct_orders:
```

Comment:
The description `all ancestors of snowplow_sessions and fct_orders` still feels a bit vague, this could be either a union or an intersection (based on the implicit set logic of English prepositional phrases).

At the risk of being too verbose, I would make this even more explicit to get the point home:
```suggestion
Run snowplow_sessions, all ancestors of snowplow_sessions, fct_orders, and all ancestors of fct_orders:
```